### PR TITLE
reorganized list of apps, added 2017 PLOS paper

### PR DESCRIPTION
**Pull request by dkp**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -105,27 +105,34 @@ <h1 class="brand-heading">Brain&nbsp;Imaging Data&nbsp;Structure</h1>
...
         <ul>
-          <li><a href="http://preprocessed-connectomes-project.org/quality-assessment-protocol/">QAP</a> (quality assesment)</li>
-          <li><a href="http://mriqc.readthedocs.org/">MRIQC</a> (quality assesment)</li>
-          <li><a href="https://github.com/poldracklab/preprocessing-workflow">FMRIPREP</a> (preprocessing workflow)</li>
-          <li><a href="https://github.com/rhodricusack/automaticanalysis">Automatic Analysis</a> (fMRI processing toolbox)</li>
-          <li><a href="https://github.com/INCF/openfmri2bids">OpenfMRI2BIDS</a> (converter)</li>
+          <li><a href="http://bids-apps.neuroimaging.io">BIDS Apps</a> (a growing set of portable containerized data processing pipelines that understand BIDS datasets)</li>
+          <li>Converters</li><ul>
+          <li><a href="https://github.com/nih-fmrif/BIDS-tools">AFNI BIDS-tools</a> (set of tools for converting to and analyzing BIDS datasets in AFNI)</li>
           <li><a href="https://github.com/INCF/BIDS2ISATab">BIDS2ISATab</a> (converter)</li>
```
Comment (chrisfilo) [inline index.html]: Adding sub lists is a great idea. I think we can drop the now redundant descriptions in brackets such as "(converter)".

File: index.html [hunk 2]
```diff
@@ -105,27 +105,34 @@ <h1 class="brand-heading">Brain&nbsp;Imaging Data&nbsp;Structure</h1>
...
+          <li><a href="http://openfmri.org">OpenfMRI</a> (repository)</li>
+          </ul>
+          <li>Quality Assessment</li><ul>
+          <li><a href="http://mriqc.readthedocs.org/">MRIQC</a> (quality assesment)</li>
+          <li><a href="http://preprocessed-connectomes-project.org/quality-assessment-protocol/">QAP</a> (quality assesment)</li>
         </ul>
-        <p>Another good introduction to the BIDS standard can be found in the <a href="http://www.nature.com/articles/sdata201644">paper published in Nature Scientific Data</a>.</p>
+        <br>
+        <p>A good introduction to the BIDS standard can be found in the <a href="http://www.nature.com/articles/sdata201644">paper published in Nature Scientific Data</a>.<br>
+        A description of how to build BIDS containerized apps can be found in the <a href=" http://doi.org/10.1371/journal.pcbi.1005209">paper published in PLOS Computational Biology</a>.</p>
```
Comment (chrisfilo) [inline index.html]: Personally I feel that "how to build containerized apps supporting BIDS inputs" sounds better.

**Comment by chrisfilo** [comment]
Thank you! This is great - I left some minor comments.

**Comment by dkp** [comment]
made changes as requested.

**Comment by chrisfilo** [comment]
Awesome! Thanks!

**Pull request merged**